src/drawing/DrawingPathGroup.scss

```diff
@@ -0,0 +1,12 @@
+$ba-DrawingTarget-transitionDelay: 25ms;
```

Comment:
Do these need to be variables if they're only used once? If so, should they be renamed?